<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #191a26; /* —Ñ–æ–Ω –¥–ª—è –≤—Å–µ–≥–æ –ø—Ä–∏–ª–æ–∂–µ–Ω–∏—è */
	font-family: &quot;GOST Common&quot;;
	text-align: center;
    color: #eeeeee;            /* —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    font-size: 14px;           /* –°—Ç–∞–Ω–¥–∞—Ä—Ç–Ω—ã–π —Ä–∞–∑–º–µ—Ä —Ç–µ–∫—Å—Ç–∞ */
}

/* –ù–∞—Å—Ç—Ä–æ–π–∫–∏ –¥–ª—è –∫–Ω–æ–ø–æ–∫ */
QPushButton {
    background-color: #7189e4;     /* –û—Å–Ω–æ–≤–Ω–æ–π —Ü–≤–µ—Ç */
    color: white;                  /*—Ç–µ–∫—Å—Ç */
    border-radius: 10px;           /* –°–∫—Ä—É–≥–ª–µ–Ω–Ω—ã–µ —É–≥–ª—ã */
    padding: 8px 16px;             /* –û—Ç—Å—Ç—É–ø—ã */
    border: 2px;     /* –ë–æ—Ä–¥–µ—Ä —Å —è—Ä–∫–∏–º –∞–∫—Ü–µ–Ω—Ç–æ–º */
}

QPushButton:hover {
    background-color: #5e71ba;     /* –Ø—Ä–∫–∏–π —Ä–æ–∑–æ–≤—ã–π —Ü–≤–µ—Ç –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
}

QPushButton:pressed {
    background-color: #e471c3;     /* –ö—Ä–∞—Å–Ω—ã–π —Ü–≤–µ—Ç –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */
	border: 2px solid #191a26;}

/* –ù–∞—Å—Ç—Ä–æ–π–∫–∏ –¥–ª—è –ø–æ–ª–µ–π –≤–≤–æ–¥–∞ —Ç–µ–∫—Å—Ç–∞ */
QLineEdit {
    background-color: #191a26;     /* –¢–µ–º–Ω—ã–π —Ñ–æ–Ω */
    color: #eeeeee;                  /* –ë–µ–ª—ã–π —Ç–µ–∫—Å—Ç */
    border: 1px solid #7189e4;     /* –§–∏–æ–ª–µ—Ç–æ–≤–∞—è —Ä–∞–º–∫–∞ */
    padding: 6px;
    border-radius: 6px;
}

QLineEdit:focus {
    border: 1px solid #e471c3;     /* –Ø—Ä–∫–∏–π —Ä–æ–∑–æ–≤—ã–π –ø—Ä–∏ —Ñ–æ–∫—É—Å–µ */
}
/* –ù–∞—Å—Ç—Ä–æ–π–∫–∏ –¥–ª—è —á–µ–∫–±–æ–∫—Å–æ–≤ */
QCheckBox {
    color: #eeeeee;

}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
    border-radius: 6px;
    background-color: #191a26;
    border: 2px solid #7189e4;     /* –ö—Ä–∞—Å–Ω–∞—è —Ä–∞–º–∫–∞ */
}

QCheckBox::indicator:checked {
    background-color: #e471c3;     /* –§–∏–æ–ª–µ—Ç–æ–≤—ã–π —Ü–≤–µ—Ç –ø—Ä–∏ –≤—ã–±–æ—Ä–µ */
    border: 2px solid #aaaaff;     /* –Ø—Ä–∫–∞—è —Ä–æ–∑–æ–≤–∞—è —Ä–∞–º–∫–∞ –ø—Ä–∏ –≤—ã–±–æ—Ä–µ */
}

</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="ServerSettingsLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="addrLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST Common</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="PrtLbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST Common</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverPortLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="UserSettingsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="shwPasswdBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST Common</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üëÅ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pollCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>polling server</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="networkModeCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>auto server search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
